vars/perfSearchBestConfigParallelPipeline.groovy

```diff
@@ -86,6 +89,9 @@ def call(Map pipelineParams) {
             booleanParam(name: 'use_client_encryption',
                          defaultValue: false,
                          description: 'Enable client encryption')
+            booleanParam(name: 'use_prepared_loaders',
+                         defaultValue: true,
+                         description: 'Run c-s process in docker (False), on prepared instance(True)')
```

Comment:
not sure that 'use_prepared_loaders' is clear enough, I would change it to something like 'run_cs_on_loader' . WDYT?
and I would change the description to:
'Run c-s commands directly on the loader (True) or as a process in a docker (False)'

Reply:
this is the term from SCT itself, we would need to change it there too.